lib/App.js

```diff
@@ -2394,7 +2397,6 @@ class App {
     appPath = path.join(appPath, appJson.id);
 
     try {
-      const stat = await statAsync(appPath);
       throw new Error(`Path ${appPath} already exists`);
     } catch (err) {
       if (err.code === undefined) throw err;
```

Comment:
This change is not right. I believe this is checking whether a folder exists in the strangest way possible:
`statAsync` throws an error if the given path doesn't exist so, if it does the next line throws.
errors thrown by Node.JS build-in modules have a `.code` property so we re-throw if the error did not come from `statAsync`.

Just call `fs.existsSync` and throw the error if it doesn't exist...

Reply:
You are completely right in both ways: 1) the change is incorrect and 2) it worked in the strangest way possible.